hammett/types.py

```diff
@@ -16,6 +17,14 @@
 
 CheckUpdateType = tuple[object, ConversationKey, BaseHandler[Update, CCT], object]
 
+
+class Document(TypedDict):
+    """The class implements the Document type."""
+
+    document_data: FileInput
+    document_name: str
```

Comment:
It's better to shorten it to `data` and `name` since the type has already got the name `Document`.